assets/build/print.css

```
@@ -0,0 +1,181 @@
+}
+.entry-content dl {
+  margin-bottom: 1em;
+}
+.entry-content dl dt {
+  float: left;
+  display: inline-block;
+  font-weight: 400;
+  text-align: center;
+  padding: 4px 9px;
+  margin-bottom: 1em;
+  font-size: 18px;
```

Stylelint (17.5.0)

[error] Unexpected duplicate "font-size" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)